notebooks/space2stats_api_demo_aggregate.ipynb

```diff
@@ -0,0 +1,221 @@
+       "1  866a4a01fffffff  {'type': 'Point', 'coordinates': [34.732706798...   \n",
+       "2  866a4a04fffffff  {'type': 'Point', 'coordinates': [38.771304043...   \n",
+       "3  866a4a057ffffff  {'type': 'Point', 'coordinates': [41.630359262...   \n",
+       "4  866a4a05fffffff  {'type': 'Point', 'coordinates': [39.038040116...   \n",
+       "\n",
+       "   sum_pop_2020  \n",
+       "0    676.912804  \n",
+       "1    347.182722  \n",
+       "2    285.943490  \n",
+       "3    279.463912  \n",
+       "4    235.258474  "
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Define the Request Payload\n",
+    "request_payload = {\n",
+    "    \"aoi\": aoi,\n",
+    "    \"spatial_join_method\": \"centroid\",\n",
+    "    \"fields\": [\"sum_pop_2020\"], \n",
+    "    \"geometry\": \"point\"\n",
+    "}\n",
+    "\n",
+    "# Get Summary Data\n",
+    "response = requests.post(SUMMARY_ENDPOINT, json=request_payload)\n",
+    "if response.status_code != 200:\n",
+    "    raise Exception(f\"Failed to get summary: {response.text}\")\n",
+    "\n",
+    "summary_data = response.json()\n",
+    "df = pd.DataFrame(summary_data)\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Aggregate Results\n",
+    "\n",
+    "In some cases, the analyst and researchers may be interested only in the summary statistics for the entire region of interest. In this section, we demonstrate how users can easily aggregate the values of the hexagons for their region of interest.\n",
+    "\n",
+    "\tWe strongly recommend being mindful of data that may be lost in this two-step aggregation process (raw data to H3 hexagons to aggregate information). Since each hexagon is given equal weight during aggregation, this can introduce inaccuracies for certain variables. For example, aggregating population data may be reasonable because each hexagon contributes equally to the total population. However, for other variables, such as ratios (e.g., the ratio of men to women), this approach may produce skewed results. The aggregated value may differ significantly compared to what you would get by aggregating the raw data directly. In such cases, it’s important to assess whether the variable should be aggregated at the hexagon level or if a different approach would yield more accurate results.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mean population per hexagon: 2865.592428932978\n",
+      "Total population for region: 73949478.22104442\n"
+     ]
+    }
+   ],
+   "source": [
+    "mean_sum_pop_2020 = df['sum_pop_2020'].mean()\n",
+    "total_pop_2020 = df['sum_pop_2020'].sum()\n",
```

Comment:
Users could calculate other variables (standard deviation, median, quantiles, or anything they need) with the functions already supported in the Python ecosystem. Moreover, they could filter some of these values based on their needs. If we provide the aggregation directly with the API, an end-user wouldn't be able to backtrack those results. It seems like a lot of complexity to add to the API for not a ton of added friction.